src/entities/Proposal/jobs.ts

```diff
@@ -172,12 +172,21 @@ async function prepareProposalsAndBudgetsUpdates(
   }
 }
 
+function logFinishableProposals(finishableProposals: ProposalAttributes[]) {
+  if (isProdEnv()) {
+    logger.log(`Finishable proposals:`)
+    finishableProposals.forEach((proposal, index) => {
+      console.log(`${index}: ${proposal.id} - ${proposal.title}`)
+    })
+  }
+}
+
 export async function finishProposal() {
   logger.log(`Running finish proposal job...`)
 
   try {
     const finishableProposals = await ProposalModel.getFinishableProposals()
-    logger.log(`Finishable proposals`, finishableProposals)
+    logFinishableProposals(finishableProposals)
```

Comment:
I think we can remove this log.

Reply:
Didn't we looked for a finished proposal recently? (the one that never finished). Maybe it's still useful 🤷